css/src/assets/styles/styles.css

```diff
@@ -0,0 +1,162 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  background-color: #F5F5F5;
+  background-size: 414px 347px;
+}
+#mainDiv {
+  margin: 12px 8px;
+  background-color: #FFF;
+  border-radius: 7px;
+  width: 394px;
+  box-shadow: 0 5px 3px #DDDDDD;
+  font-family: museo-sans, Sans-Serif;
+}
+
+/** Images and Icons **/
+.getLoose {
+  position: relative;
+  margin-top: -40px;
+  padding: 12px 12px 0;
+  width: 370px;
+  height: auto;
+}
+.clockIcon {
+  position: relative;
+  top: 2px;
+  width: 16px;
+  height: auto;
+}
+.songLength {
+  color: #A6A6A6;
+  position: relative;
+  bottom: 1px;
+}
+.shoeIcon {
+  position: relative;
+  width: 30px;
+  height: auto;
+  padding-right: 5px;
+  top: 10px;
+}
+#instructions {
+  position: relative;
+  bottom: 10px;
+  color: #A6A6A6;
+  padding: 5px 0 5px 7px;
+  background-color: #FFF;
+  transition: .2s;
+  margin-left: 10px;
+  z-index: 200;
+}
+
+/***** Heart Container Stuff ******/
+#heartContainer {
+  visibility: hidden;
+  width: 39px;
+  height: 39px;
+  background-color: #F5F5F5;
+  position: relative;
+  z-index: 10;
+  left: 330px;
+  top: 24px;
+  border-radius: 100%;
+}
+#heartContainer.active {
+  visibility: visible;
+}
+#heartContainer.clicked {
+  background-color: rgba(230, 38, 45, 1);
+}
+#outlinedHeart {
+  position: relative;
+  z-index: 20;
+  left: 11px;
+  top: 11px;
+  width: 16px;
+  height: auto;
+}
+#outlinedHeart.active {
+  visibility: hidden;
+}
+#solidHeart {
+  visibility: hidden;
+  position: relative;
+  z-index: 30;
+  right: 9px;
+  top: 11px;
+  width: 16px;
+  height: auto;
+}
+#solidHeart.active {
+  visibility: visible;
+}
+#circleBorder {
+  visibility: hidden;
+  z-index: 1;
+  position: relative;
+  width: 20px;
+  height: 20px;
+  border: 8px solid rgba(230, 38, 45, 1);
+  border-radius: 100%;
+  margin-top: -19px;
+  margin-left: 2px;
+}
+#circleBorder.active {
+  visibility: visible;
+  animation-duration: .4s;
+  animation-name: ripple;
+}
+
+@keyframes ripple {
+  0% {
+  }
+  80% {
+    margin-top: -38px;
+    margin-left: -18px;
+    padding: 20px;
+  }
+  100% {
+    border: 8px solid rgba(230, 38, 45, 0);
+  }
+}
+
+#instructions.active {
+  margin-left: 90px;
+}
+hr {
+  color: #F5F5F5;
+  border-top-style: none;
+  border-left-style: none;
+}
+div ul:nth-of-type(1) {
+  padding: 5px 12px 17px;
+}
+div ul:nth-of-type(2) {
+  padding: 10px 12px 9px 0;
+}
+div ul:nth-of-type(2) li:nth-of-type(1) {
+  padding-left: 11px;
+  position: absolute;
+  margin-top: 5px;
+  color: #A6A6A6;
+}
+.slideInText {
+  padding: 10px 12px 9px 16px;
+}
+li {
```

Comment:
Styling HTML element directly many times leads to having to override this styles. I'd recommend using a `class` instead.